### PR TITLE
add docs for making image formulas using temporary blob storage

### PR DESCRIPTION
**Pull request by spencer-codaio**
adding some docs per Slack on utilizing the new post-processing work for images in formulas in conjunction with temporary blob storage.

### CODE REVIEW

File: documentation/examples/image/boring_avatar.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import * as coda from "@codahq/packs-sdk";
+export const pack = coda.newPack();
+
+pack.addNetworkDomain("source.boringavatars.com");
```
Comment (ekoleda-codaio) [inline documentation/examples/image/boring_avatar.ts:4]: It's a best practice to use the highest level domain you can. Change this to `boringavatars.com`.

File: documentation/examples/image/boring_avatar.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+pack.addFormula({
+  name: "BoringAvatar",
+  description: "Get a boring avatar image.",
+  parameters: [
+    coda.makeParameter({
+      type: coda.ParameterType.Number,
+      name: "Size",
```
Comment (ekoleda-codaio) [inline documentation/examples/image/boring_avatar.ts:12]: I've been using all lowercase letters for parameter names. Change this to `size` for consistency.

File: documentation/examples/image/boring_avatar.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  execute: async function ([size], context) {
+    let resp = await context.fetcher.fetch({ 
+      method: "GET", 
+      url: `https://source.boringavatars.com/beam/${size}`,
+    });
+    // This API returns direct SVG code used to generate the avatar.
+    let svg = resp.body;
+
+    let url = await context.temporaryBlobStorage
+                      .storeBlob(Buffer.from(svg), "image/svg+xml");
```
Comment (ekoleda-codaio) [inline documentation/examples/image/boring_avatar.ts:27]: Optional: Use `isBinaryResponse: true` in the fetch request and skip the `Buffer.fom()` here.

File: documentation/scripts/documentation_config.ts [hunk 4]
```diff
@@ -718,9 +725,16 @@ export const Examples: Example[] = [
         content: 'A formula that return an external image. This sample returns a random photo of a cat.',
         codeFile: './examples/cats/formula.ts',
       },
+      {
+        name: 'Image result from temporary URL',
+        content:
+          'A formula that returns an image uploaded to `temporaryBlobStorage`. This sample returns a random avatar using an API that returns SVG code used to generate an avatar. You could also imagine locally creating an SVG or other image and uploading it to `temporaryBlobStorage`.',
```
Comment (ekoleda-codaio) [inline documentation/scripts/documentation_config.ts:731]: What do you mean "locally" in this context? On the local machine or in the Pack code itself? May be worth rephrasing to make that clearer.

**Review by ekoleda-codaio — CHANGES_REQUESTED** [review]
Thanks so much! A few small comments.

**Comment by spencer-codaio** [comment]
@ekoleda-codaio updated!

**Review by ekoleda-codaio — APPROVED** [review]

**Pull request merged**